### ISSUE TITLE
Non-owners are not allowed to read certain value fields

### ISSUE THREAD

**Issue by SamueleDassatti** [IglooCloud/IglooBering#177]
The following `value` fields return "You are not allowed to perform this operation" if an admin, editor or spectator tries to fetch them:
- hidden
- cardSize
- name
- updatedAt
- createdAt
- myRole
- device

**Comment by ZaninAndrea** [comment]
@SamueleDassatti cannot reproduce, can you give more details as to when the bug occurs? 

Tried creating a device, claiming it on an enviroment, adding a FloatValue to it, sharing the environment with another user and fetching 

```graphql
{
  floatValue(id: "{{floatValueId}}") {
    hidden
    cardSize
    name
    updatedAt
    createdAt
    myRole
    device {
      id
    }
  }
}
```

and 

```graphql
{
  device(id:"{{deviceId}}"){
    values(limit:5){
      hidden
      cardSize
      name
      updatedAt
      createdAt
      myRole
      device{id}
    }
  }
}
```

**Comment by ZaninAndrea** [comment]
Solved by eed3e0ab20327508791b44c12286afb5f2375d90, device.values was listing private values also for non-producer users
